src/actions/pause.js

```diff
@@ -104,9 +104,8 @@ export function paused(pauseInfo: Pause) {
       dispatch(evaluateExpressions());
     }
 
-    dispatch(
-      selectSource(frame.location.sourceId, { line: frame.location.line })
-    );
+    const { line, column } = frame.location;
+    dispatch(selectSource(frame.location.sourceId, { line, column }));
```

Comment:
shouldn't this be `dispatch(selectSource(frame.location.sourceId, { location: frame.location }));`